packages/lib-classifier/src/plugins/tasks/survey/components/SurveyTask.spec.js

```diff
@@ -15,19 +15,20 @@ describe('SurveyTask', function () {
       const DefaultStory = composeStory(Default, Meta)
 
       it('should show a filter button', function () {
-        const { getByRole } = render(<DefaultStory />)
+        render(<DefaultStory />)
         
         // filterButton is the Filter button above the choices
-        const filterButton = getByRole('button', { name: 'SurveyTask.CharacteristicsFilter.filter' })
+        const filterButton = screen.getByLabelText('SurveyTask.CharacteristicsFilter.filter')
         
         expect(filterButton).to.be.ok()
       })
 
       it('should show the choices', function () {
-        const { getAllByRole } = render(<DefaultStory />)
+        render(<DefaultStory />)
         
+        const choicesMenu = screen.getByTestId('choices-menu')
```

Comment:
I think there's only one menu, so this can be:
```suggestion
        const choicesMenu = document.querySelector([role=menu])
```

Reply:
`querySelector` might be faster. I'm not sure.